### DISCUSSION TITLE
Livewire Feature wire:navigate not in Flux UI

### DISCUSSION THREAD

**Discussion by shadwattai** [livewire/flux#942] [Feature Requests]
Hello team,

I've tried to work with the components, but I can't find the usual link with **wire:navigate** in the docs.
Has it been replaced? How do I achieve the benefits equivalent to that with flux components?

Thank you for your feedback.

**Comment by jeffchown** [comment]
@shadwattai Not quite sure what you are looking for.  `wire:navigate` is a Livewire feature (and as you know is documented on the Livewire site: https://livewire.laravel.com/docs/wire-navigate) and should work with any Blade markup/component that contains a link/href, this of course includes Flux components.

e.g.

```
<flux:navbar>
    <flux:navbar.item href="/home" icon="home" wire:navigate>Home</flux:navbar.item>
    <flux:navbar.item href="/features" icon="puzzle-piece" wire:navigate>Features</flux:navbar.item>
    <flux:navbar.item href="/pricing" icon="currency-dollar" wire:navigate>Pricing</flux:navbar.item>
    <flux:navbar.item href="/about" icon="user" wire:navigate>About</flux:navbar.item>
</flux:navbar>
```

**Comment by joshhanley** [comment]
@shadwattai yeah @jeffchown is correct, `wire:navigate` is a Livewire feature, hence why it's not documented in the Flux docs. Flux links/ nav items/ etc. should work with `wire:navigate`. So going to close this. Please feel free to open a new issues or feature request if you think something is missing.
